### DISCUSSION TITLE
How to use higher version of Cypress?

### DISCUSSION THREAD

**Discussion by pietrodito** [Appsilon/rhino#503] [Q&A]
When I install Rhino from CRAN, Cypress is v7. I can see that last release is v13.

Is there a way to upgrade cypress once a directory has been `rhino::init`(ed)?

**Comment by jakubnowicki** [comment]
Hi @pietrodito, we are in the process of updating Cypress to the new version. You can check [this Pull Request](https://github.com/Appsilon/rhino/pull/504) for more details.
In an existing project, you can update Cypress by changing the version in `.rhino/package.json` and then running `npm install cypress`. Please note that the `.rhino` directory will be added to your project only after you run one of the functions that use Node (e.g., `rhino::build_sass`).
After upgrading Cypress, run `rhino::test_e2e(interactive = TRUE)` and follow the migration wizard.
Lastly, by default, the `.rhino` directory is gitignored, so to share the changes, you will need to modify the `.rhino/.gitignore` file.
